wallet/cardano-sl-wallet.cabal

```diff
@@ -98,6 +98,9 @@ library
                       Pos.Util.BackupPhrase
                       Pos.Util.Mnemonics
 
+                      -- shared test sources needed by tests of wallet-new
+                      Test.Pos.Wallet.Web.Mode
```

Comment:
I think that - considering what I have said above about the `DevelopmentSpec` relying too much on the internals of the old wallet, we should move the `DevelopmentSpec` here and just reuse the code in `wallet-new`. This way we clearly separate the concerns and the day the `wallet` project will be decommissioned we will be forced to revisit our development specs anyway, and rewrite them with the new wallet code. What do you think?

Reply:
> relying too much on the internals of the old wallet

Again, that's we currently do with most of all `V1` endpoints. As I've understood that's ok, because the _first_ step is to wrap `V0` endpoints with an `V1` interface (API) and the _second_ step is to re-write the internal thing to be independent from `V0`. Or do I miss something here? Feel free to discuss that in Slack if there are more information to share.

Reply:
> Or do I miss something here?

Well, there is a substantial difference in using old code within some internal handler vs writing test code which is coupled with it, as in the former case you can test the V1 endpoint as a blackbox (thus completely independent from how the internal implementation is delivered) whereas in the second case you will have to tear down the entire test code once you get rid of the old `wallet` code. But for now let's be pragmatic, as it would be overkill spend more time on these internal endpoints.

> Please note that DevelopmentSpec is testing functions of wallet-new, especially deleteSecretKeys

You're right, I didn't appreciate we were querying the handler directly -- in which case, you made the right decision and I'm happy with the status quo, apologies 😉